sql/core/src/test/scala/org/apache/spark/sql/UDFSuite.scala

```diff
@@ -58,6 +61,17 @@ class UDFSuite extends QueryTest {
     ctx.dropTempTable("tmp_table")
   }
 
+  test("SPARK-8005 input_file_name") {
```

Comment:
now we moved the way we define input_file_name and spark_partition_id, maybe we can remove these test cases from Hive.